project.php

```diff
@@ -399,6 +399,21 @@ function do_project_info_table()
     echo_row_a( _("Author"),          $project->authorsname );
     echo_row_a( _("Language"),        $project->language );
     echo_row_a( _("Genre"),           _($project->genre) );
+
```

Comment:
Let's move this back down to where it was prior. This bit of info isn't super useful to most proofers whereas some of the other bits below it are.